<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Main_Form</class>
 <widget class="QWidget" name="UI_Main_Form">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Common_Button" name="btnImport">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>open background image(&amp;O)</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnLeftToRight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>left to right</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="btnRightToLeft">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>right to left</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="Common_Button" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>about(&amp;H)...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnStretch">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>stretch draw</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Common_Button</class>
   <extends>QPushButton</extends>
   <header>state_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
